libcst/codemod/commands/convert_precent_format_to_fstring.py

```diff
@@ -0,0 +1,121 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
Does this need a FB copyright? I have a TODO on my plate to add a copyright/preamble check to the lint job so I don't keep forgetting.

Reply:
Yes, every Python file needs a copyright unless it's whitelisted.
Add a check in the lint is a good idea for getting feedback early while developing.

Reply:
I created #227 for this, since we don't have lint open sourced yet.